### PR TITLE
Small plugin for jump. Also adds check for NNN_PIPE to autojump plugin

### PR DESCRIPTION
**Pull request by snide**
Hello! Thank you for this fantastic little program. I like using [jump](https://github.com/gsamokovarov/jump), which is very similar to auto-jump. Figured I'd add a small plugin for it.

Here is a small mp4 showing it working (exactly the same as autojump). I'm primarily a designer, so yell if I missed something!

https://snid.es/2020AUG/GlygMMHKZTW9hBrU.mp4

### CODE REVIEW

**Review by KlzXS — APPROVED** [review]
@jarun we should add the `NNN_PIPE` check in `autojump` plugin as well.

**Comment by snide** [comment]
@KlzXS Added it for you.

File: plugins/autojump [hunk 1]
```diff
@@ -9,6 +9,11 @@
 # Shell: POSIX compliant
 # Author: Marty Buchaus
 
+if [ -z "$NNN_PIPE" ]; then
+  echo 'ERROR: NNN_PIPE is not set' | ${PAGER:-less}
```
Comment (jarun) [inline plugins/autojump:13]: I think the indent is 2 chars, we use 4. Can you please update it in both the plugin changes?

**Review by jarun — CHANGES_REQUESTED** [review]

File: plugins/autojump (hunk 1 above)
Comment (snide) [reply plugins/autojump:13]: Absolutely! My bad.

Comment (jarun) [reply plugins/autojump]: No problem! You got most of the things right! Even I miss sometimes. ;)

Comment (snide) [reply plugins/autojump]: Done.

**Comment by jarun** [comment]
Thank you for your contribution!

**Comment by jarun** [comment]
I missed something... why not just merge these 2 plugins into one?

Keep jump as default and autojump as fallback.

**Comment by jarun** [comment]
We do combine near-similar-functionality utilities in a single plugin.

**Comment by jarun** [comment]
@snide you can follow the `fzopen` plugin to combine these 2 into a single one.

**Comment by jarun** [comment]
Took care of it.

**Comment by snide** [comment]
@jarun Ah. Thanks. That makes sense. Good to know for next time!

**Pull request merged**